transmart-data/ddl/postgres/biomart_user/views/patient_set_bitset.sql

```diff
@@ -1,9 +1,13 @@
 --
 -- Name: patient_set_bitset; Type: VIEW; Schema: biomart_user; Owner: -
 --
-CREATE VIEW biomart_user.patient_set_bitset AS
+CREATE OR REPLACE VIEW biomart_user.patient_set_bitset AS
 SELECT
-  collection.result_instance_id AS result_instance_id,
```

Comment:
@ewelinagr does this change address https://jira.thehyve.nl/browse/TMT-375? Or you meant something broader?

Reply:
Partially yes, I updated the description of TMT-375